problem_builder/mentoring.py

```diff
@@ -817,13 +858,9 @@ def author_edit_view(self, context):
         """
         Add some HTML to the author view that allows authors to add child blocks.
         """
-        fragment = Fragment(u'<div class="mentoring">')  # This DIV is needed for CSS to apply to the previews
-        self.render_children(context, fragment, can_reorder=True, can_add=False)
-        fragment.add_content(u'</div>')
-        fragment.add_content(loader.render_template('templates/html/mentoring_add_buttons.html', {}))
-        fragment.add_content(loader.render_template('templates/html/mentoring_url_name.html', {
```

Comment:
Nit: I'm not sure how important this is, but I think we can put the URL name back in to make it consistent with Step Builder block (see screenshot).

![screen shot 2016-09-29 at 10 08 54](https://cloud.githubusercontent.com/assets/32585/18945902/d61b8a0c-862c-11e6-9cfc-6f69a157aaca.png)